src/tribler/ui/src/dialogs/SaveAs.tsx

```diff
@@ -268,6 +270,12 @@ export default function SaveAs(props: SaveAsProps & JSX.IntrinsicAttributes & Di
                     </label>
                 </div>
                 <DialogFooter>
+                    <div className="flex flex-row-reverse text-muted-foreground space-x-2">
+                        {(warning === undefined) ? <></> : <AlertTriangle className="self-center" />}
+                    </div>
+                    <div className="flex flex-row-reverse text-muted-foreground space-x-2">
+                        {(warning === undefined) ? <></> : <label className="whitespace-pre-line text-xs self-center">{warning}</label>}
+                    </div>
```

Comment:
Maybe it's written like this for a reason, but I feel the following is a bit more readable:

```typescript
                    {warning && (
                        <div className="flex flex-row text-muted-foreground space-x-2">
                            <AlertTriangle className="self-center" />
                            <label className="whitespace-pre-line text-xs self-center">{warning}</label>
                        </div>
                    )}
```

Reply:
There is no reason beyond it working at all. I adopted your approach 👍